nilearn/_utils/data_gen.py

```diff
@@ -193,25 +263,52 @@ def generate_labeled_regions(shape, n_regions, rand_gen=None, labels=None,
     return Nifti1Image(data, affine)
 
 
-def generate_labeled_regions_large(shape, n_regions, rand_gen=None,
+def generate_labeled_regions_large(shape,
+                                   n_regions,
+                                   random_state=0,
                                    affine=np.eye(4)):
     """Similar to generate_labeled_regions, but suitable for a large number of
     regions.
 
     See generate_labeled_regions for details.
+
+    Parameters
+    ----------
+    shape : tuple
+        Shape of returned array.
+
+    n_regions : int
+        Number of regions to generate. By default (if "labels" is None),
```

Comment:
Probably copy pasted from `generate_labeled_regions` since there is no `labels` parameter in this function.